docs/specs/discovery.md

```diff
@@ -24,7 +24,9 @@ follow the appropriate [_discovery event schema_](../../gencode/docs/events_disc
 The overall discovery sequence involves multiple components that work together to provide the overall flow:
 * **Devices**: The target things that need to be discovered, configured, and ultimately communicate point data.
 * **Spotter**: Operative node that performs _discovery_, scanning local networks and producing observations.
-* **Agent**: Cloud-based agent responsible for managing the overall _discovery_ and _mapping_ process (how often, what color, etc...).
+* **Registry**: Digital Twin of the on-prem devices on the cloud for the particular site.
```

Comment:
I'd leave out `Registry` -- it's not adding anything of value. Technically, _everything_ goes through the registry so the lines that don't stop there are inaccurate!